packages/app/src/cli/services/logs.test.ts

```diff
@@ -193,6 +202,64 @@ describe('logs', () => {
       storeNameById: expectedStoreMap,
     })
   })
+
+  test('should call ensureDevContext with customLogsInfoBox flag and render custom info box', async () => {
+    // Given
+    const sources = ['extensions.source']
+    await setupDevContext(sources)
+    vi.mocked(storeFromFqdn).mockResolvedValueOnce(testOrganizationStore({shopId: '2', shopDomain: 'other-fqdn'}))
+
+    // When
+    await logs({
+      reset: false,
+      format: 'text',
+      directory: 'directory',
+      apiKey: 'api-key',
+      storeFqdns: ['store-fqdn', 'other-fqdn'],
+      sources,
+      status: 'status',
+      configName: 'config-name',
+      userProvidedConfigName: 'user-provided-config-name',
+    })
+
+    // Then
+    expect(ensureDevContext).toHaveBeenCalledWith({
+      apiKey: 'api-key',
+      configName: 'config-name',
+      customInfoBox: true,
```

Comment:
When testing for a specific thing, I like to pull it out to the `// Given` area to really highlight it.
To make it clearer that this is really what we're focused on, I'd add a `const customInfoBox = true` or something similar above.

Just a nit, feel free to disregard.